crates/flox-rust-sdk/src/flox.rs

```diff
@@ -543,6 +543,9 @@ impl Flox {
             ..Default::default()
         };
 
+        let mut extra_args = vec!["--quiet".to_string(), "--quiet".to_string()];
```

Comment:
If 1 is default, why not just one quiet? https://nixos.org/manual/nix/stable/command-ref/opt-common.html

Reply:
I think one wasn't quite quiet enough
But please don't refrain from trying again with one quiet

Reply:
I think the docs are wrong (I might create a PR)